package.json

```diff
@@ -74,8 +74,9 @@
     "ajv": "^4.7.1",
     "append-query": "2.0.1",
     "axe-core": "^2.3.1",
-    "babel-core": "^6.24.1",
+    "babel-core": "^6.26.3",
```

Comment:
This upgrade looks okay. Check out the [babel changelog](https://github.com/babel/babel/blob/6.x/CHANGELOG.md) for specifics.